docs/server/source/simple-network-setup.md

```diff
@@ -194,14 +217,37 @@ persistent_peers = "<Member 1 node id>@<Member 1 hostname>:46656,\
 <Member N node id>@<Member N hostname>:46656,"
 ```
 
-Now the Member must run two commands:
-- `bigchaindb start`
-- `tendermint node`
+## Member: Start MongoDB, BigchainDB and Tendermint
+
+To start MongoDB, one uses the command `mongod` but that will run it in the foreground. If you want to run it in the background (so it will continue running after you logout), you can use `mongod --fork --logpath /var/log/mongodb.log`. (You might have to create the `/var/log` directory if it doesn't already exist.)
+
+The _recommended_ approach is to reboot the machine and let the MongoDB startup script start MongoDB. Don't do that yet though…
+
+To start BigchainDB, one uses the command `bigchaindb start` but that will run it in the foreground. If you want to run it in the background (so it will continue running after you logout), you can use `nohup`, `tmux`, or `screen`. For example, `nohup bigchaindb start 2>&1 > bigchaindb.log &`
+
+The _recommended_ approach is to create a startup script for BigchainDB, so it will start right after the boot of the operating system. (As mentioned earlier, MongoDB should already have a startup script.)
+
+To start Tendermint, one uses the command `tendermint node` but that will run it in the foreground. If you want to run it in the background (so it will continue running after you logout), you can use `nohup`, `tmux`, or `screen`. For example, `nohup tendermint node 2>&1 > tendermint.log &`
+
+The _recommended_ approach is to create a startup script for BigchainDB, so it will start right after the boot of the operating system. (As mentioned earlier, MongoDB should already have a startup script.)
```

Comment:
This seems to repeated, any particular reason its this way?

Reply:
Oh, good catch. It should say "...for Tendermint, so it will start..." I will fix that.